operations/patch_util.go

```diff
@@ -120,15 +103,43 @@ func (p *patchParams) createPatch(ac *legacyClient, conf *ClientSettings, diffDa
 		syncStatuses:      p.SyncStatuses,
 		syncTimeout:       p.SyncTimeout,
 		finalize:          p.Finalize,
+		backport:          p.Backport,
 	}
 
 	newPatch, err := ac.PutPatch(patchSub)
 	if err != nil {
 		return nil, err
 	}
+
+	return newPatch, nil
+}
+
+func (p *patchParams) validateSubmission(diffData *localDiff) error {
```

Comment:
It looks like this function is only ever called at the same time as createPatch; why did you separate it out?

Reply:
This was an evolution:
I originally separated it out because I wanted to avoid validation for backport commits (because we used to validate the description wasn't empty). I then separated handling the description into its own function, but I left this separate because it seemed conceptually distinct from uploading a patch.